Index: Dockerfile
==================================================================
--- Dockerfile
+++ Dockerfile
@@ -73,13 +73,13 @@
 ## ---------------------------------------------------------------------
 ## STAGE 2: Pare that back to the bare essentials.
 ## ---------------------------------------------------------------------
 
 FROM scratch
-WORKDIR /jail
+WORKDIR /
 ARG UID=499
-ENV PATH "/bin:/usr/bin:/jail/bin"
+ENV PATH "/bin:/usr/bin"
 
 ### Lay BusyBox down as the first base layer. Coupled with the host's
 ### kernel, this is the "OS."
 COPY --from=builder /tmp/bbx/busybox /bin/
 COPY --from=builder /etc/os-release /etc/
@@ -101,13 +101,13 @@
     && mknod -m 666 dev/null    c 1 3                                  \
     && mknod -m 444 dev/urandom c 1 9
 
 ### Do Fossil-specific things atop those base layers; this will change
 ### as often as the Fossil build-from-source layer above.
-COPY --from=builder /tmp/fossil bin/
+COPY --from=builder /tmp/fossil /usr/bin/
 RUN set -x                                                             \
-    && ln -s /jail/bin/fossil /usr/bin/f                               \
+    && ln -s /usr/bin/fossil /usr/bin/f                                \
     && echo -e '#!/bin/sh\nfossil sha1sum "$@"' > /usr/bin/sha1sum     \
     && echo -e '#!/bin/sh\nfossil sha3sum "$@"' > /usr/bin/sha3sum     \
     && echo -e '#!/bin/sh\nfossil sqlite3 --no-repository "$@"' >      \
        /usr/bin/sqlite3                                                \
     && chmod +x /usr/bin/sha?sum /usr/bin/sqlite3
@@ -118,10 +118,9 @@
 ## ---------------------------------------------------------------------
 
 EXPOSE 8080/tcp
 CMD [ \
     "bin/fossil", "server", \
-    "--chroot", "/jail",    \
     "--create",             \
     "--jsmode", "bundled",  \
     "--user", "admin",      \
     "museum/repo.fossil"]

